extensions/standard-processors/tests/unit/YamlConnectionParserTest.cpp

```diff
@@ -163,10 +163,10 @@ TEST_CASE("Connections components are parsed from yaml", "[YamlConfiguration]")
             "flowfile expiration: \n"
             "drop empty: \n"});
         YamlConnectionParser yaml_connection_parser(connection_node, "test_node", parent_ptr, logger);
-        CHECK_THROWS(yaml_connection_parser.getWorkQueueSizeFromYaml());
-        CHECK_THROWS(yaml_connection_parser.getWorkQueueDataSizeFromYaml());
-        CHECK_THROWS(yaml_connection_parser.getFlowFileExpirationFromYaml());
-        CHECK_THROWS(yaml_connection_parser.getDropEmptyFromYaml());
+        REQUIRE(0 == yaml_connection_parser.getWorkQueueSizeFromYaml());
+        REQUIRE(0 == yaml_connection_parser.getWorkQueueDataSizeFromYaml());
+        REQUIRE(0 == yaml_connection_parser.getFlowFileExpirationFromYaml());
+        REQUIRE(0 == yaml_connection_parser.getDropEmptyFromYaml());
```

Comment:
I would leave these as `CHECK(...)`.  In general, `CHECK` is preferable to `REQUIRE`: if eg. the second and fourth tests fail, then with `CHECK`, we will get this info at once, but with `REQUIRE`, we will only know on the first run that the second test failed, then we'll have to fix it and rerun the test to find out that the fourth test fails, too.

Reply:
I agree, updated in 3d60c2431ccbd6073868aba88924e94d16867b80

Reply:
Also disabled `readability/check` linter rule in 3e4ccdb98e53fbd2b49176eb3f32d86bb0cb06d0 as it suggested using CHECK_EQ instead of CHECK, but catch2 does not have a CHECK_EQ assertion.